internal/pipeline/testdata/app-with-generator/configs/db/Kptfile

```diff
@@ -0,0 +1,20 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: db
+pipeline:
+  mutators:
+  - image: gcr.io/sunilarora-sandbox/httpbin-fn:latest
```

Comment:
I plan to move this function to function catalog under `contrib` section. This will point to the function from the catalog before this PR is merged, so will keep this comment unresolved for now.

Reply:
Initiated a PR for adding this function in function catalog: https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/182